src/tests/ftest/util/security_test_base.py

```diff
@@ -70,6 +70,20 @@ def acl_principal(usergroup, name):
         entry = "u:" + name + "@"
     return entry
 
+def get_user_type(test_user):
+    """Get test user's user type for ACE access control entry.
```

Comment:
```suggestion
    """Get test user's user type for ACL access control entry.
```
I don't recognize ACE...

Reply:
DAOS ACL (access control list) contains a list of individual ACE (access control entry).

Reply:
I see. Thanks.